mne/io/kit/kit.py

```diff
@@ -804,6 +804,11 @@ def read_raw_kit(input_fname, mrk=None, elp=None, hsp=None, stim='>',
     raw : Instance of RawKIT
         A Raw object containing KIT data.
 
+    Notes
```

Comment:
Technically, `Notes` ]should come after] (https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) `See Also`, but we don't need to hold up the PR for that